core/http-auth-aws/src/main/java/software/amazon/awssdk/http/auth/aws/internal/signer/DefaultAwsV4HttpSigner.java

```diff
@@ -127,6 +148,10 @@ private static V4PayloadSigner v4PayloadSigner(
             throw new UnsupportedOperationException("Unsigned payload is not supported with event-streaming.");
         }
 
+        if (isChunkEncoding) {
+            return new AwsChunkedV4PayloadSigner(properties.getCredentialScope(), 1024 * 128);
```

Comment:
Where does this value come from?

Reply:
default chunk size from old code - I'll make it a static value up top.